src/components/Reports/GDPRPermissions/PermissionExpandableRow.js

```diff
@@ -130,16 +138,17 @@ const PermissionExpandableRow = ({ field, email, dependencies: { dopplerApiClien
                       </td>
                     </tr>
                   ) : (
-                    permissions.map(({ value, originIP, date }, index) => {
+                    permissions.map(({ value, originIP, date, originType }, index) => {
                       return (
-                        <tr>
+                        <tr key={index}>
                           <td>
-                            <PermissionValue key={index} value={value} />
+                            <PermissionValue value={value} />
                           </td>
                           <td>{originIP}</td>
                           <td>
-                            <FormattedDate key={index} value={date} />
+                            <FormattedDate value={date} />
                           </td>
+                          {originType ? <td>{originType}</td> : null}
```

Comment:
In this case I'd show a text like: `none`/`vacío` or maybe an empty string instead `null`

Reply:
As the comment before, if this is something that we don't have from the BE yet, I think that this is ok like that.